modules/covid_vaccine/app/services/covid_vaccine/v0/enrollment_service.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'net/sftp'
+require 'covid_vaccine/v0/expanded_registration_submission_csv_generator'
+
+module CovidVaccine
+  module V0
+    class EnrollmentService
+      def initialize
+        @records = CovidVaccine::V0::ExpandedRegistrationSubmission.enrolled_us.order(:created_at)
+        @timestamp = Time.now.utc.strftime('%Y%m%d%H%M%S')
+      end
+
+      def send_enrollment_csv
+        Net::SFTP.start(sftp_host, sftp_username, password: sftp_password) do |sftp|
+          sftp.upload!(csv_as_io, remote_file_path, name: file_name, progress: EnrollmentHandler.new)
+        end
+      end
+
+      private
+
+      def csv_as_io
+        @io ||= ExpandedRegistrationSubmissionCSVGenerator.new(@records).io
+      end
+
+      # TODO: TBD
+      def remote_file_path
+        "/#{file_name}"
+      end
+
+      def file_name
```

Comment:
Consider prefixing "TEST_" to the filename if this is not from prod? 
(Can't use RAILS_ENV but could consider checking on `Settings.hostname == "api.va.gov" `
or `Settings.vsp_environment == "production"`

Reply:
Could be a later optimization but would be good for when we're passing files around.

Reply:
I think I'd prefer to just make the filename something you could override in that context